pkgs/os-specific/linux/autosuspend/default.nix

```diff
@@ -0,0 +1,56 @@
+{ lib,
+  fetchFromGitHub,
+  python3
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "autosuspend";
+  version = "4.0.0";
+
+  src = fetchFromGitHub {
+    owner = "languitar";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "03qca6avn7bwxcavif7q2nqfzivzp0py7qw3i4hsb28gjrq9nz36";
+  };
+
+  propagatedBuildInputs = with python3.pkgs; [
+    portalocker
+    psutil
+    dbus-python
+  ];
+
+  checkInputs = with python3.pkgs; [
+    pytestCheckHook
+    python-dbusmock
+    pytest-httpserver
+    pytest-cov
```

Comment:
```suggestion
```
please patch out the usage of cov. We can't send the data and collecting it slows down the tests.